jasper_server/jasper_yaml_object.py

```diff
@@ -33,5 +33,7 @@ class jasper_yaml_object(osv.osv):
         'offset': fields.integer('Offset'),
         'limit': fields.integer('Limit'),
         'order': fields.char('Order', size=128),
+        'yaml_context': fields.char('Context',),
```

Comment:
offset and limit are without yaml_prefix. Please keep consistent with that notation.